docs/ref/models/querysets.txt

```diff
@@ -1108,6 +1108,11 @@ database.
     :class:`related managers<django.db.models.fields.related.RelatedManager>`,
     any prefetched cache for the relation will be cleared.
 
+    .. versionchanged:: 2.2
```

Comment:
I'm wondering if this is needed/right? This is a bug fix so we'll backport to v2.1, so this isn't right? Right? 😄

Reply:
Yeah, my bad! I've removed this but kept a note in the 2.2 release notes.

Reply:
I think this should be a note in the 2.1.1/txt release notes. (And not 2.2, since it will be long-in by then.)

Thanks!

Reply:
Done! Thanks 👍

Reply:
If it's not a regression from a previous release or a major bug in a new feature it won't be backported. See our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions). For 2.2, bug fixes don't need release notes.

Reply:
Grrr. Eventually I will get the hang of this. 😀. Thanks Tim.